common/changes/@uifabric/experiments/fix-button-global-styling_2019-05-15-18-24.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "<Fabric> element does not affect global button styling.",
+      "type": "minor"
```

Comment:
Shouldn't these be of type `patch` since this is not changing the API surface?

Reply:
I think `minor` is appropriate as it will affect global styles for anyone that uses the `<Fabric>` element.